### DISCUSSION TITLE
`keybind-megalist` in the wiki defines `neorg_leader` variable, but it's not being used.

### DISCUSSION THREAD

**Discussion by vitaly** [nvim-neorg/neorg#219] [Q&A]
and since it's a local one, I can't understand how it's supposed to work.

Also, the example only has 5 key bindings. as the homepage says "Keybinds that make logical sense. Simply think, don't remember.", I was expecting a bit more ;). the example says 'This is a massive code block that...', so I wander if it did have more in the past but may be stuff was removed..

**Comment by max397574** [comment]
There are a lot of keybindings for new stuff which isn't in master yet and the docs aren't all up to date

**Reply by vhyrro** [reply]
Damn, beat me to it by a little lmao

**Comment by vhyrro** [comment]
There are a bunch more, they're just not up to date in the keybind megalist. Well be working on an automatic documentation generator to allow it to just autoupdate :)

I'm a bit occupied with treesitter stuff right now, but after that I'll be focusing on both the documentation generator and max's journal module PR. Hopefully it should be figured out fairly soon!

**Comment by max397574** [comment]
this is what there is atm
https://github.com/nvim-neorg/neorg/blob/new-links/lua/neorg/modules/core/keybinds/default_keybinds.lua

**Comment by vitaly** [comment]
that's not on the master, but I can wait till it's merged ;)..

now... I have another problem with the mappings. some weird interaction with https://github.com/folke/which-key.nvim
once `which-key` kicks in, it removes all the `norg` maps. if I try to manually set the same kind of map in `core.keybinds.events.enable_keybinds` it remains, and not affected by `which-key`. so, it seems that the map is different somehow.
When I try to look at the mappings with, say, `:map gtd`, I see that the map is not `<buffer>` scoped, and yet, it's not available on a buffer that is not `norg`. I'd appreciate some help with what's going on here...

**Reply by max397574** [reply]
I use which-key and I have no problems
I think that the mapping not work in other buffers is expected
even though I'd like to use especially the gtd capture mapping in other buffers as well

**Reply by vhyrro** [reply]
It's an issue that I'm aware of on the `main` branch. In unstable this has been fixed. Basically the old implementation used global mappings because of some bugs present in Neovim at the time. Since those are gone now though it's been replaced with buffer local mappings in the aforementioned branch. We'll be merging everything into `main` soon - it's ironic that `unstable` is more stable than `main` hah, sorry about that.

**Comment by vitaly** [comment]
the unstable branch seems to define some bindings that do not exist. e.g. `core.gtd.base.*`. Do I need to install some additional modules like with telescope?

**Reply by vitaly** [reply]
ah. found. the `gtd` module isn't loaded by default. well, in that case it shouldn't setup bindings ;)

**Reply by danymat** [reply]
@vhyrro ?

**Comment by hardfau1t** [comment]
but how do i set the leader key? is it just like `neorg_leader.."gtd"`
